src/server/api/trpc.ts

```diff
@@ -107,7 +107,7 @@ export const publicProcedure = t.procedure;
 
 /** Reusable middleware that enforces users are logged in before running the procedure. */
 const enforceUserIsAuthed = t.middleware(({ ctx, next }) => {
-  if (!ctx.session || !ctx.session.user) {
+  if (!ctx.session) {
```

Comment:
I think I need some feedback what happened here

Reply:
I was an unnecessary check. It came with the t3 boilerplate. Since `user` is a property of `session` and we check if `session` or `session.user` exist. It was just bad logic.